<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>syotaPin</class>
 <widget class="QDialog" name="syotaPin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Syötä tunnuslukusi</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textPinkoodi">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>145</width>
     <height>41</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="logIn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="N1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>45</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="N2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>45</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="N3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>45</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="N6">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>45</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="N5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>45</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="N4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>45</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="N9">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>220</y>
     <width>45</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="N8">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>45</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="N7">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>45</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="N0">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>45</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>45</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="GoBackPin">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>370</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Takaisin</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
